examples/google-internal-load-balancing/README.md

```diff
@@ -0,0 +1,38 @@
+# Internal Load Balancing in Google Cloud
+
+This provides a template for setting up internal load balancing in Google Cloud. It directly mirrors the tutorial in the [GCP Internal Load Balancing Documentation](https://cloud.google.com/compute/docs/load-balancing/internal/).
+
+To run the example, you'll need to [download your credentials from Google Cloud Console](https://www.terraform.io/docs/providers/google/#credentials). A suggested path for the downloaded file is `~/.gcloud/Terraform.json`.
```

Comment:
@danawillow, what do you think about having the example use Cloud Shell, and not requiring credential download?